<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Mukul" failures="1" tests="4" name="class_60_IRetryAnalyzer_DataDrivenTesting.Amazon_Login_UsingAssertion" time="47.593" errors="0" timestamp="2025-05-01T19:50:46 IST" skipped="3">
  <testcase classname="class_60_IRetryAnalyzer_DataDrivenTesting.Amazon_Login_UsingAssertion" name="amazon_login" time="10.233">
    <skipped/>
  </testcase> <!-- amazon_login -->
  <testcase classname="class_60_IRetryAnalyzer_DataDrivenTesting.Amazon_Login_UsingAssertion" name="amazon_login" time="11.583">
    <skipped/>
  </testcase> <!-- amazon_login -->
  <testcase classname="class_60_IRetryAnalyzer_DataDrivenTesting.Amazon_Login_UsingAssertion" name="amazon_login" time="14.326">
    <failure message="Sorry ! the title are not matching expected [Amazon Sign In] but found [Online Shopping site in India: Shop Online for Mobiles, Books, Watches, Shoes and More - Amazon.in]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Sorry ! the title are not matching expected [Amazon Sign In] but found [Online Shopping site in India: Shop Online for Mobiles, Books, Watches, Shoes and More - Amazon.in]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1590)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at class_60_IRetryAnalyzer_DataDrivenTesting.Amazon_Login_UsingAssertion.amazon_login(Amazon_Login_UsingAssertion.java:33)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:569)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:269)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:76)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- amazon_login -->
  <testcase classname="class_60_IRetryAnalyzer_DataDrivenTesting.Amazon_Login_UsingAssertion" name="amazon_login" time="11.451">
    <skipped/>
  </testcase> <!-- amazon_login -->
</testsuite> <!-- class_60_IRetryAnalyzer_DataDrivenTesting.Amazon_Login_UsingAssertion -->
